src/button/high-visibility.scss

```diff
@@ -10,22 +11,26 @@
     color: t(iui-color-foreground-accessory);
   }
 
-  > .iui-icon {
+  &[disabled] {
+    @include iui-button-disabled;
+  }
+
+  > .iui-button-icon {
     @include themed {
       fill: t(iui-color-foreground-accessory);
     }
   }
 
-  &:enabled:hover,
-  &:enabled:active,
+  &:hover,
+  &:active,
   &:focus-visible {
```

Comment:
unrelated to your changes focus-visible should be removed from this list to fix Safari/IE bug. https://css-tricks.com/one-invalid-pseudo-selector-equals-an-entire-ignored-selector/

Suggest changing it to just `:focus` and then adding a separate `:focus:not(:focus-visible)` to negate those styles.

Reply:
Fixed focus, not-focus-visible code.